basedtyping/__init__.py

```diff
@@ -556,3 +558,27 @@ def f[T](t: TypeForm[T]) -> T: ...
                             
                              reveal_type(f(int | str))  # int | str
                          """)
+
+# generic aliases
+if TYPE_CHECKING:
+    dict_keyiterator = _collections_abc.dict_keyiterator  # type: ignore[attr-defined]
+    dict_valueiterator = _collections_abc.dict_valueiterator  # type: ignore[attr-defined]
+    dict_itemiterator = _collections_abc.dict_itemiterator  # type: ignore[attr-defined]
+    list_iterator = _collections_abc.list_iterator  # type: ignore[attr-defined]
+    list_reverseiterator = _collections_abc.list_reverseiterator  # type: ignore[attr-defined]
+    set_iterator = _collections_abc.set_iterator  # type: ignore[attr-defined]
+    tuple_iterator = _collections_abc.tuple_iterator  # type: ignore[attr-defined]
```

Comment:
these shouldnt be here because they dont exist. if theres a reason to assume theyll exist in the future there should be a comment saying so